### ISSUE TITLE
[Issue] VSSTester doesn't truncate the database logs

### ISSUE THREAD

**Issue by worldsdream** [microsoft/CSS-Exchange#2046]
**Describe the issue**
Using the latest version of VSSTester.ps1 (# Version 23.12.06.1742)

**Expected behavior**
VSSTester.ps1 doesn't truncate the database logs. There are no errors in the scripts. Everything seems to be running correctly from start to finish, but it isn't truncating...

I enabled circular logging and dismounted/mounted the database and all the logs cleared successfully.

Normally, VSSTester.ps1 would truncate the logs without unmounting/mounting the database.

**Comment by dpaulson45** [comment]
It will truncate the logs, if there are enough logs to even truncate. It sounds like you don't have any logs to truncate at the time you run VSSTester.ps1 because they are not committed to the database yet (aka Check Point Depth). That is why it will truncate once you dismount and remount the database, because we did state that we backed up the logs and we can remove them once they are committed to the database, which occurs at dismount and remount in this scenario. 

In a non-production environment, no active users/mail flow, it is very common to see up to 1000 logs not to be truncated due to the lack of activity and the checkpoint depth is that large.
